server/src/ClientToDBAPI/Address/mainRoute.ts

```diff
@@ -1,64 +1,70 @@
 import { Router, Request, Response } from 'express';
 
-import logger from '../../Logger/Logger';
 import City from '../../Models/Address/City';
 import Street from '../../Models/Address/Street';
 import Country from '../../Models/Address/Country';
-import { graphqlRequest } from '../../GraphqlHTTPRequest';
 import { Severity } from '../../Models/Logger/types';
+import { errorStatusCode, graphqlRequest } from '../../GraphqlHTTPRequest';
 import {GET_ALL_CITIES, GET_CITY_STREETS, GET_ALL_COUNTRIES} from '../../DBService/Address/Query';
+import logger, { invalidDBResponseLog, validDBResponseLog, launchingDBRequestLog} from '../../Logger/Logger';
 
 const addressRoute = Router();
 
 addressRoute.get('/cities', (request: Request, response: Response) => {
-  graphqlRequest(GET_ALL_CITIES, response.locals).then((result: any) => {
-    let cities: City[] = [];
-    if (result && result.data && result.data.allCities) {
-      cities = result.data.allCities.nodes.map((city: City) => ({
-        id: city.id,
-        displayName: city.displayName,
-      }));
-    } 
-      return response.send(cities);
-  });
+    const citiesLogger = logger.setup({
+      workflow: 'query all cities',
+      investigation: response.locals.epidemiologynumber,
+      user: response.locals.user.id
+    });
+    citiesLogger.info(launchingDBRequestLog(), Severity.LOW);
+    graphqlRequest(GET_ALL_CITIES, response.locals)
+    .then((result: any) => {
+        citiesLogger.info(validDBResponseLog, Severity.LOW);
+        const cities: City[] = result?.data?.allCities?.nodes || [];
+        response.send(result?.data?.allCities?.nodes || []);
```

Comment:
What's the importance of cities constant if you're sending the result like this?